packages/react-button/src/components/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -251,8 +247,8 @@ exports[`Button renders anchor when href prop is provided 1`] = `
         --color-brand-borderColor: transparent;
         --color-brand-checked-background: #005a9e;
         --color-brand-checked-contentColor: #ffffff;
-        --color-brand-checkedHovered-background: #005a9e;
-        --color-brand-checkedHovered-contentColor: #ffffff;
+        --color-brand-checked-hovered-background: #005a9e;
```

Comment:
this does not seem right. I think the states should all be at the same level.

Reply:
Fixed!